.github/workflows/push.yaml

```diff
@@ -189,7 +189,7 @@ jobs:
           customLink: 'https://obsproject.com/'
 
       - name: Upload Artifacts 📡
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
```

Comment:
Right now the sparkle artifact relies on the fact that v3 artifacts are combined server side go produce a single output with both architectures. So just switching to v4 will not work.

Reply:
I can revert this specific change for now, but we'll want to address that "soon", per GitHub's current plan to "transition all actions to run on Node 20 by Spring 2024".

Reply:
Reverted this line.

Reply:
Yeah that's fine, we can probably just split them up as well, having them be one single artifact was convenient but not technically necessary.